assets/src/edit-story/elements/video/display.js

```diff
@@ -29,16 +27,13 @@ function VideoDisplay( props ) {
 }
```

Comment:
^ Probably the `x` and `y` etc. properties should be removed above in the `VideoDisplay` as well? Currently, it's adding all properties to `Element` ( `...props` ), this still includes these, even if the `propTypes` were updated, correct?

Reply:
Yeah. Generally these attributes are just element attributes. The template may or may not use them. But template should declare what it needs. So, yeah, I removed these from the declaration.